packages/angular_devkit/build_angular/src/utils/build-options.ts

```diff
@@ -13,6 +13,7 @@ import {
   Budget,
   CrossOrigin,
   ExtraEntryPoint,
+  I18NDuplicateTranslation,
```

Comment:
@alan-agius4 @petebacondarwin do you notice what I am missing here?
It seems that changing schema and running `yarn build` generated schema types that are missing both `I18NDuplicateTranslation` and `I18NMissingTranslation`.

Reply:
Since the enum values of `I18NDuplicateTranslation` and `I18NMissingTranslation` are the same. They have been merged into a single type.

```ts
    /**
     * How to handle duplicate translations for i18n.
     */
    i18nDuplicateTranslation?: I18NTranslation;
    /**
     * How to handle missing translations for i18n.
     */
    i18nMissingTranslation?: I18NTranslation;
```